submissions/Mifaresss/task_js_oop/app.js

```diff
@@ -0,0 +1,136 @@
+const CELL_WIDTH = 101;
+const CELL_HEIGHT = 83;
+const COLUMNS = 5;
+const ROWS = 6;
+const FIELD_WIDTH = CELL_WIDTH * COLUMNS;
+const FIELD_HEIGHT = CELL_HEIGHT * ROWS;
+
+const PLAYER_START_POSITION_X = 202;
+const PLAYER_START_POSITION_Y = 385;
+const PLAYER_STEP_LENGTH_X = 101;
+const PLAYER_STEP_LENGTH_Y = 83;
+const AVAILABLE_PLACES_FOR_PLAYER = {
+	minX: PLAYER_START_POSITION_X - PLAYER_STEP_LENGTH_X * 2,
+	maxX: PLAYER_START_POSITION_X + PLAYER_STEP_LENGTH_X * 2,
+	minY: PLAYER_START_POSITION_Y - PLAYER_STEP_LENGTH_Y * 5,
+	maxY: PLAYER_START_POSITION_Y
+}
+const PLAYER_WIDTH = 77;
+const PLAYER_HEIGHT = 66;
+
+const ENEMIES_SPEED = { min: 100, max: 350 };
+const ENEMIES_START_POSITION_X = -100;
+const ENEMIES_START_POSITION_Y = {
+	first: 60,
+	second: 143,
+	third: 226,

```

Comment:
Hmmm... JS god gave us arrays so that we do not need to write anything alike `array[first], array[second]... array[fiftySixth]`

Reply:
done